public/css/index.css

```diff
@@ -7,209 +7,152 @@
     --rounded: 999px;
 }
 
-body {
-    background-color: var(--navy-blue) !important;
-}
-
-body div, body img {
+div, img, a, article, label, .post-panel {
     border-radius: 8px;
 }
 
-body a {
-    border-radius: 8px;
+a:hover {
+    text-decoration: underline !important;
 }
 
-body textarea {
-    line-height: 30px;
-    letter-spacing: 2px;
+textarea {
+    line-height: 30px !important;
+    letter-spacing: 2px !important;
+    resize: none;
+    border-radius: 8px;
 }
 
-body p {
+p {
     line-height: 30px;
     letter-spacing: 2px;
     white-space: pre-line
 }
 
-body button {
-    border-radius: 8px;
+button {
+    border-radius: 8px !important;
     color: white;
 }
 
-body button:hover {
+button:hover {
     opacity: 0.8;
 }
 
-body a {
+a {
     text-decoration: none !important;
     color: black;
 }
 
+/* 方形 */
+.square {
+    border-radius: 0;
+}
+
+/* 左下圓角 */
+.rounded-bl {
+    border-radius: 0 0 0 8px;
+}
+
+/* 右下圓角 */
+.rounded-br {
+    border-radius: 0 0 8px 0;
+}
+
+.bg-blue {
+    background-color: #002C55;
+}
+
+.round-icon {
+    height: 48px;
+    width: 50px;
+    border-radius: 22px;
+    margin: auto 10px auto 10px;
+}
+
+.btn-hover:hover {
```

Comment:
You could add hover to elements by tailwind class as well. Check [here](https://tailwindcss.com/docs/hover-focus-and-other-states#hover-focus-and-active)